src/ServerRequestFactory.php

```diff
@@ -256,4 +263,48 @@ private static function marshalHttpsValue($https) : bool
 
         return 'on' === strtolower($https);
     }
+
+    /**
+     * @param string|list<string> $host
+     * @return array Array of two items, host and port, in that order (can be
+     *     passed to a list() operation).
+     */
+    private static function marshalHostAndPortFromHeader($host)
```

Comment:
since it is private modifier, return type declaration can be added.

Reply:
Fixed - thanks!